arangograph/scim.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+description: How to enable SCIM provisioning
+title: SCIM Provisioning
+---
+# SCIM Provisioning
+
+{% hint 'info' %}
+This feature is only available in ArangoGraph Enterprise.
+{% endhint %}
+
+ArangoGraph provides support to control and manage members access in
+ArangoGraph organizations with the
+**System for Cross-domain Identity Management** (SCIM) provisioning. 
+This enables you to propagate to ArangoGraph any user access changes by using
+the dedicated API.
+
+{% hint 'info' %}
+To enable the SCIM feature, get in touch with the ArangoGraph team via an
+ArangoGraph Support Ticket.
+{% endhint %}
+
+## About SCIM
+
+[SCIM](https://www.rfc-editor.org/rfc/rfc7644){:target="_blank"}, or the System
+for Cross-domain Identity Management [specification](http://www.simplecloud.info/){:target="_blank"},
+is an open standard designed to manage user identity information.
+SCIM provides a defined schema for representing users, and a RESTful
+API to run CRUD operations on these user resources.
+
+The SCIM specification expects the following operations so that the SSO system
+can sync the information about user resources in real time:
+
+- `GET /Users` - List all users.
+- `GET /Users/:user_id` - Get details for a given user ID.
+- `POST /Users` - Invite a new user to ArangoGraph.
+- `PUT /Users/:user_id` - Update a given user ID.
+- `DELETE /Users/:user_id` - Delete a specified user ID.
+
+ArangoGraph organization administrators can generate an API key for a specific organization.
+The API token consists of a key and a secret. Using this key and secret as the
+Basic Authentication Header (Basic Auth) in SCIM provisioning, you can access the APIs and
+manage the user resources.
```

Comment:
It will be good to mention here that don't set time to live here else you will have to keep rotating basic auth credentials in okta.
<img width="1510" alt="Screenshot 2023-02-14 at 4 56 08 PM" src="https://user-images.githubusercontent.com/6964334/218723863-6c3cb9ca-ecea-4500-8d07-39e198483a18.png">